src/helpers.js

```diff
@@ -50,6 +50,13 @@ let helpers = {
     return truthTest(params, bodies, context, (left, right) => {
       return left <= right;
     });
+  },
+  action(context, params, bodies) {
+    let {selector, type, method} = params;
+    type = type || 'click';
+    let frag = bodies.main(context);
+    frag.querySelector(selector).addEventListener(type, context[method].bind(context));
```

Comment:
Let me know if this would be useless, but maybe you could allow binding to a different context:

let boundContext = params.context || context;
frag.querySelector(selector).addEventListener(type, context[method].bind(boundContext));